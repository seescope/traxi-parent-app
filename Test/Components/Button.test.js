import 'react-native';
import React from 'react';
import AndroidButton from '../../App/Components/Button.android.js';

import renderer from 'react-test-renderer';

jest.mock('react-native-material-kit');

it('renders on Android correctly', () => {
  const onPress = () => {};
  const tree = renderer.create(
    <AndroidButton onPress={onPress}>Testing!</AndroidButton>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
