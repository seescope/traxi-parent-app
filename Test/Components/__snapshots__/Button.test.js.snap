exports[`test renders on Android correctly 1`] = `
<View>
  <View
    onPress={[Function]}
    style={
      Object {
        "backgroundColor": "#448AFF",
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFF",
          "fontWeight": "normal",
        }
      }>
      TESTING!
    </Text>
  </View>
</View>
`;
